core/src/main/scala/org/genivi/sota/core/DeviceUpdatesResource.scala

```diff
@@ -62,9 +65,15 @@ class DeviceUpdatesResource(db: Database,
 
   protected lazy val updateService = new UpdateService(DefaultUpdateNotifier, deviceRegistry)
 
+  protected val publishFn = MessageBusManager.start(system, ConfigFactory.load()) match {
```

Comment:
It's not clear to me how I would use getOrElse without throwing away the ConfigException that start() can return.